docs/UserGuide.md

```diff
@@ -202,14 +324,19 @@ _Details coming soon ..._
 
 ## Command summary
 
-| Action     | Format, Examples                                                                                                                                                                                             |
```

Comment:
You remember to update the list as well. Good job!

Reply:
thanks!